src/public/sass/pages/_check-your-answers.scss

```diff
@@ -0,0 +1,83 @@
+// Recommended - Use these styles for the check your answers pattern
```

Comment:
Why? Shouldn't we use styles from GOV.UK frontend?

Reply:
From my understanding the GOV.UK frontend does not have all styles, both extensions and the prototype make use of these 'custom' ones, and so for the purposes of matching the prototype exactly, I have just copied and pasted their styles. Regarding scope, it might be worth deleting anything we don't use yet (as currently it is mostly for formating the summary tables).